languages/wikipron/src/README.md

```diff
@@ -0,0 +1,77 @@
+"Big scrape" scripts
+====================
+
+[scrape.py](scrape.py) calls WikiPron's scraping functions on all
+Wiktionary languages with over 100 entries.
+[generate\_summary.py](generate_summary.py) generates a [README](../README.md)
+and a [TSV](../languages_summary.tsv) with selected
+information regarding the contents of the TSVs [scrape.py](scrape.py)
+generated and the configuration settings
+that were passed to scrape. [remove\_duplicates.sh](remove_duplicates.sh)
+sorts and removes entries in each TSV if they have 
+the same graphemic form and phonemic form as a previous entry.
+ [languages.json](languages.json) provides
+[scrape.py](scrape.py) with a dictionary containing the information it
+needs to call scrape on all Wiktionary languages with over 100 entries 
+and is also used to generate the previously mentioned [README](../README.md).
+[codes.py](codes.py) is used to generate
+[languages.json](languages.json). It queries Wiktionary for all languages
+with over 100 entries. It also outputs
+[unmatched\_languages.json](unmatched_languages.json), a list of languages on
+Wiktionary that have over 100 entries but that could not be matched with an ISO
+639 language code.
+
+Steps used to update the dataset
+--------------------------------
+
+1.  Run [codes.py](codes.py) to update
+    [languages.json](languages.json).
+    -   If there are new Wiktionary languages with over 100 entries, they will
+        be added to [languages.json](languages.json).
+    -   As mentioned in the comment at the top of [codes.py](codes.py),
+        any new languages added to [languages.json](languages.json) will
+        not have their case-folding settings specified. 
+        Their `"casefold"` value will therefore be set to `None`.
+    -   Whether or not to apply case-folding for these new languages needs
+        to be manually set by changing the "casefold" value within
+        [languages.json](languages.json).
+    -   To find new languages you can run`git diff languages.json` 
+        or search for `null` values within 
+        [languages.json](languages.json).
+2.  Run [scrape.py](scrape.py).
+    -   By default `cut_off_date` in `main()` is set using
+        `datetime.date.today().isoformat()` but can be set manually 
+        using an ISO formatted string (ex. "2019-10-31").
+3.  Run [serbo\_croat\_split.py](serbo_croat_split.py).
```

Comment:
Wonder if we should give this a name like `split_hbs.py` or something. Is there any other language that might need this?

Reply:
The only thing I've come across is that the [last](https://en.wiktionary.org/wiki/%E1%9C%80#Tagalog) Wiktionary Tagalog entry is written in Baybayin script, which we scrape and is included [in our Tagalog TSV](https://github.com/kylebgorman/wikipron/blob/master/languages/wikipron/tsv/tgl_phonemic.tsv#L1391).

It's possible other languages use two scripts, but I haven't come across them yet.

Reply:
I suggest making the file names `split_*.py` then so if there are more of them, they sort nicely.